apertium-spa-cat.spa-cat.dix

```diff
@@ -57428,6 +57428,511 @@ Cal desambiguar si és topònim. Hi ha moltíssims casos en què cal: el Fondo>e
+<e>   <p><l>azadón<s n="n"/></l>  <r>aixada<s n="n"/></r></p></e>
+<e>   <p><l>babucha<s n="n"/></l>   <r>babutxa<s n="n"/></r></p></e>
+<e>   <p><l>bacaladero<s n="n"/></l>  <r>bacallaner<s n="n"/></r></p></e>
+<e>   <p><l>bagatela<s n="n"/></l>  <r>bagatel·la<s n="n"/></r></p></e>
+<e>   <p><l>bajel<s n="n"/></l>   <r>vaixell<s n="n"/></r></p></e>
+<e>   <p><l>bajeza<s n="n"/></l>  <r>baixesa<s n="n"/></r></p></e>
+<e>   <p><l>ballestero<s n="n"/></l>  <r>ballester<s n="n"/></r></p></e>
+<e>   <p><l>banqueta<s n="n"/></l>  <r>banquet<s n="n"/></r></p></e>
+<e>   <p><l>barbasco<s n="n"/></l>  <r>candelera<s n="n"/></r></p></e>
+<e>   <p><l>barbilampiño<s n="adj"/></l>  <r>barbamec<s n="adj"/></r></p></e>
+<e>   <p><l>barbillera<s n="n"/></l>  <r>estopada<s n="n"/></r></p></e>
+<e>   <p><l>barboquejo<s n="n"/></l>  <r>barballera<s n="n"/></r></p></e>
+<e>   <p><l>barcia<s n="n"/></l>  <r>garbelladures<s n="n"/></r></p></e>
+<e>   <p><l>barquero<s n="n"/></l>  <r>barquer<s n="n"/></r></p></e>
+<e>   <p><l>barquilla<s n="n"/></l>   <r>barqueta<s n="n"/></r></p></e>
+<e >  <p><l>barrenero<s n="n"/></l>   <r>barrinaire<s n="n"/></r></p></e>
+<e>   <p><l>barrilete<s n="n"/></l>   <r>barrilet<s n="n"/></r></p></e>
+<e>   <p><l>bastonazo<s n="n"/></l>   <r>bastonada<s n="n"/></r></p></e>
+<e>   <p><l>beldad<s n="n"/></l>  <r>beutat<s n="n"/></r></p></e>
+<e>   <p><l>bellaco<s n="adj"/></l>   <r>brivall<s n="adj"/></r></p></e>
+<e>   <p><l>bergante<s n="n"/></l>  <r>bergant<s n="n"/></r></p></e>
+<e>   <p><l>bermejo<s n="adj"/></l>   <r>vermell<s n="adj"/></r></p></e>
+<e>   <p><l>besamanos<s n="n"/></l>   <r>besamans<s n="n"/></r></p></e>
+<e>   <p><l>bibliófilo<s n="n"/></l>  <r>bibliòfil<s n="n"/></r></p></e>
+<e>   <p><l>bienaventurado<s n="adj"/></l>  <r>benaurat<s n="adj"/></r></p></e>
+<e>   <p><l>billonario<s n="adj"/></l>  <r>bilionari<s n="adj"/></r></p></e>
+<e>   <p><l>biplaza<s n="adj"/></l>   <r>biplaça<s n="adj"/></r></p></e>
+<e>   <p><l>bisonte<s n="n"/></l>   <r>bisó<s n="n"/></r></p></e>
+<e>   <p><l>blanquinoso<s n="adj"/></l>   <r>blanquinós<s n="adj"/></r></p></e>
+<e>   <p><l>blocao<s n="n"/></l>  <r>blocau<s n="n"/></r></p></e>
+<e>   <p><l>bonete<s n="n"/></l>  <r>bonet<s n="n"/></r></p></e>
+<e>   <p><l>boquear<s n="vblex"/></l>   <r>boquejar<s n="vblex"/></r></p></e>
+<e>   <p><l>bosquecillo<s n="n"/></l>   <r>bosquet<s n="n"/></r></p></e>
+<e>   <p><l>boticario<s n="n"/></l>   <r>apotecari<s n="n"/></r></p></e>
+<e>   <p><l>breviario<s n="n"/></l>   <r>breviari<s n="n"/></r></p></e>
+<e>   <p><l>brumoso<s n="adj"/></l>   <r>boirós<s n="adj"/></r></p></e>
+<e>   <p><l>bruxismo<s n="n"/></l>  <r>bruxisme<s n="n"/></r></p></e>
+<e>   <p><l>búfalo<s n="n"/></l>  <r>búfal<s n="n"/></r></p></e>
+<e>   <p><l>bulería<s n="n"/></l>   <r>buleria<s n="n"/></r></p></e>
+<e>   <r><l>burladero<s n="n"/></l>   <r>burlador<s n="n"/></r></r></e>
+<e>   <p><l>bullir<s n="vblex"/></l>  <r>bullir<s n="vblex"/></r></p></e>
```

Comment:
"Bullir" in catalan already exists in this dictionary. So you have to either
a) Add a restriction LR, so it only works from Spanish to Catalan, or
b) Add a semantic disambiguation rule

Most probably you want to do the (a).

This will need to be done also with any other word that already existed in the dictionaries (vermell, tafaner, pipa,...)